src/Admin/FieldDescriptionInterface.php

```diff
@@ -282,6 +285,15 @@ public function getAssociationAdmin();
     // NEXT_MAJOR: Uncomment the following line
     // public function hasAssociationAdmin(): bool;
 
+    // NEXT_MAJOR: Uncomment the following line
+    // public function describesAssociation(): bool;
+
+    // NEXT_MAJOR: Uncomment the following line
+    // public function describesSingleValuedAssociation(): bool;
+
+    // NEXT_MAJOR: Uncomment the following line
+    // public function describesCollectionValuedAssociation(): bool;
```

Comment:
Why describes ? Since it returns a bool, I would expect methods to be called `hasAssociation`, ...

Should we add a comment to explain the purpose of these methods ?

Reply:
> Why describes ? Since it returns a bool, I would expect methods to be called `hasAssociation`, ...

It is what was used in the original PR and I liked, it gives some semantic.

> Should we add a comment to explain the purpose of these methods ?

I added the comments from the original PR

Reply:
If I understand correctly, the method `describesAssociation` means "Does this field description describes an association ?"

Reply:
Yes